core/tools/translations/sort_keys.py

```diff
@@ -1,18 +1,64 @@
+#!/usr/bin/env python3
 import json
+import sys
+from pathlib import Path
 
 from helpers import ALL_LANGUAGES, TRANSLATIONS_DIR
 
-for lang in ALL_LANGUAGES:
-    lang_file = TRANSLATIONS_DIR / f"{lang}.json"
-    lang_data = json.loads(lang_file.read_text())
 
-    for section_name, section in lang_data["translations"].items():
+def normalize_translations(data):
+    for section in data["translations"].values():
         for key in section:
             if "title" in key:
-                lang_data["translations"][section_name][key] = lang_data[
-                    "translations"
-                ][section_name][key].upper()
+                section[key] = section[key].upper()
```

Comment:
IIUC, this statement is not invoked on our translations (since they don't seem to have "sections"):

<img width="1932" height="1083" alt="Screenshot from 2025-07-22 09-45-10" src="https://github.com/user-attachments/assets/2b04408d-05d4-4bc7-9ee4-a3b334d8e43d" />

`section` is iterating over translated values, so `key` is iterating over the translated characters :confused: 

The code was behaving in this way from the beginning:
https://github.com/trezor/trezor-firmware/blame/b8ea21d24a35b67b3a1b2d59b9f1a96e2ad01b63/core/tools/translations/sort_keys.py

Is it still needed? Can we remove it? 
CC: @matejcik